src/devices/namron.ts

```diff
@@ -972,6 +972,24 @@ const definitions: Definition[] = [
             await reporting.currentSummDelivered(endpoint1);
         },
     },
+    {
+        zigbeeModel: ['4512761'],
+        model: '4512761',
+        vendor: 'Namron',
+        description: 'Namron ZigBee rele 16A',
+        fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering, fz.power_on_behavior],
+        toZigbee: [tz.on_off, tz.power_on_behavior],
+        exposes: [e.switch(), e.power(), e.current(), e.voltage(), e.energy(), e.power_on_behavior()],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genBasic', 'genOnOff', 'haElectricalMeasurement', 'seMetering']);
+            await endpoint.read('haElectricalMeasurement', ['acVoltageMultiplier', 'acVoltageDivisor']);
```

Comment:
I would suggest switching to use the convenience method `readEletricalMeasurementMultiplierDivisors` instead of reading each of these. 
This also doesn't report any power consumption (for me at least), so you should also add `readMeteringMultiplierDivisor`, leaving the final configure as 
```` 
    configure: async (device, coordinatorEndpoint, logger) => {
        const endpoint = device.getEndpoint(1);
        await reporting.bind(endpoint, coordinatorEndpoint, ['genBasic', 'genOnOff', 'haElectricalMeasurement', 'seMetering']);
        await reporting.readEletricalMeasurementMultiplierDivisors(endpoint);
        await reporting.readMeteringMultiplierDivisor(endpoint);
        await reporting.onOff(endpoint);
    },
```` 

That successfully reports `Power`, `Voltage`, `Current`, and `Energy`.

@enrawi will you update your PR, or want me to submit a new one?